src/cocotb/types/array.py

```diff
@@ -14,38 +14,43 @@ class Array(ArrayLike[T]):
 
     Arrays are similar to, but different from Python :class:`list`\ s.
     An array can store values of any type or values of multiple types at a time, just like a :class:`list`.
```

Comment:
```suggestion
    An array can store values of any type or values of multiple types at a time, just like a :class:`!list`.
```
Suppress excessive linking.